concordia/documents.py

```diff
@@ -15,18 +15,21 @@
 site_report = Index("site_reports")
 site_report.settings(number_of_shards=1, number_of_replicas=0)
 
+asset = Index("assets")
+asset.settings(number_of_shards=1, number_of_replicas=0)
```

Comment:
I don't think this is the time for it but we should probably review the number of shards